domains/_discord.chaotic.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ChaoticChaosTheGreat",
+        "email": "ryan.ryanabraham@gmail.com"
```

Comment:
is the email valid? i dont think thats a valid email, try removing the ryan. from it (just noticed the email thing)

Reply:
email is valid

Reply:
send a email to that to verify, actually is valid soooo sorry about this, ill merge it now